sdk/communication/communication-phone-numbers/test/public/siprouting/utils/msUserAgentPolicy.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  PipelinePolicy,
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest,
+} from "@azure/core-rest-pipeline";
+import { env } from "@azure-tools/test-recorder";
+
+export function createMSUserAgentPolicy(): PipelinePolicy {
+  return {
+    name: "msUserAgentPolicy",
+    async sendRequest(request: PipelineRequest, next: SendRequest): Promise<PipelineResponse> {
+      const useragent = env.AZURE_USERAGENT_OVERRIDE;
```

Comment:
The logic is unusual. So if no user agent is not found, we automatically assume that it's acs-test?

Reply:
Yes, because this UserAgentPolicy is only added from SDK tests, I wanted to make sure that every test run will be marked as "acs-test". 
Hardcoding values is not a good way but I think in this case there is no harm since it's only done for the tests.